dali/c_api/c_api.cc

```diff
@@ -803,3 +804,28 @@ int daliPreallocatePinnedMemory(size_t bytes) {
     return -1;
   }
 }
+
+void daliGetSerializedCheckpoint(daliPipelineHandle_t pipe_handle,
+                                 daliExternalContextCheckpoint *external_context,
```

Comment:
The external_context could be const, right? I'd add a check for it being nullptr too.

Reply:
Fixed, thank you!